enterprise/frontend/src/embedding-sdk/components/private/SdkGlobalStylesWrapper.tsx

```diff
@@ -0,0 +1,60 @@
+import { css } from "@emotion/react";
+import styled from "@emotion/styled";
+import type { HTMLAttributes } from "react";
+
+import { rootStyle } from "metabase/css/core/base.styled";
+import { defaultFontFiles } from "metabase/css/core/fonts.styled";
+import { useSelector } from "metabase/lib/redux";
+import { aceEditorStyles } from "metabase/query_builder/components/NativeQueryEditor/NativeQueryEditor.styled";
+import { getFontFiles } from "metabase/styled-components/selectors";
+import { getMetabaseCssVariables } from "metabase/styled-components/theme/css-variables";
+import { saveDomImageStyles } from "metabase/visualizations/lib/save-chart-image";
+import type { FontFile } from "metabase-types/api";
+
+interface SdkContentWrapperProps {
+  baseUrl?: string;
+}
+
+export function SdkGlobalStylesWrapper({
+  baseUrl,
+  ...divProps
+}: SdkContentWrapperProps & HTMLAttributes<HTMLDivElement>) {
+  const fontFiles = useSelector(getFontFiles);
+
+  return (
+    <SdkGlobalStylesInner
+      baseUrl={baseUrl}
+      fontFiles={fontFiles}
+      {...divProps}
+    />
+  );
+}
+
+const SdkGlobalStylesInner = styled.div<
+  SdkContentWrapperProps & {
+    fontFiles: FontFile[] | null;
+  }
+>`
+  ${({ theme }) => getMetabaseCssVariables(theme)}
```

Comment:
We need this because of components like Popovers. Popovers still need the Metabase CSS variables for their colors, so we inject the CSS variables in both `SdkGlobalStyles` (applies to Mantine components) and `PublicComponentStylesWrapper` (applies to Metabase components). [Refer to this thread for more info.](https://metaboat.slack.com/archives/C063Q3F1HPF/p1718808917198979)

Reply:
This is what happens when we only inject the CSS variables in the component wrapper. It breaks quite a lot of places.

![CleanShot 2567-06-20 at 20 55 27@2x](https://github.com/metabase/metabase/assets/4714175/dd1737cb-54e4-46d1-9c69-de2e53b557af)